catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/dialog/RegularExpressionAssistantDialogTest.java

```diff
@@ -100,6 +108,34 @@ public void testIsHtmlExtractorInList() {
 		onView(withText(R.string.formula_editor_regex_html_extractor_dialog_title)).check(matches(isDisplayed()));
 	}
 
+	@Test //refactor to wiki button or html extract once they are implemented
+	public void testIsWikiButtonInList() {
+		//setup
+		clickOnAssistantInFunctionList();
+
+		//test
+		onView(withText(R.string.formula_editor_dialog_wiki_button)).check(matches(isDisplayed()));
+	}
+
+	@Test
+	public void testOpenWikipageOnWikiButtonClick() {
+		Intents.init();
+
+		//setup
+		clickOnAssistantInFunctionList();
+
+		try {
+			//Click Catrobat RegEx Wiki Button
```

Comment:
```suggestion
```
Code is clear enough. Please limit comments to places where they are absolutely essential only.